src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintPlugin.kt

```diff
@@ -1,8 +1,11 @@
 package org.jlleitschuh.gradle.ktlint
 
+import com.android.build.gradle.BaseExtension
```

Comment:
Checked on pure Kotlin project (without any android dependencies) - works ok. When I remove this `compileOnly` dependency - plugin isn't compiled.